lang/cs/Org.Apache.REEF.Client/AzureBatch/Util/WindowsCommandBuilder.cs

```diff
@@ -48,5 +49,15 @@ protected override string GetDriverClasspath()
 
             return string.Format("'{0};'", string.Join(ClassPathSeparator, classpathList));
         }
+
+        public override string GetIpAddressFilePath()
+        {
+            return "%AZ_BATCH_JOB_PREP_WORKING_DIR%\\hostip.txt";
```

Comment:
hostip.txt to GetIpAddressFilePath()?

Reply:
I don't understand the comment. Please elaborate.

---
In reply to: [200411277](https://github.com/sharathmalladi/reef/pull/6#discussion_r200411277) [](ancestors = 200411277)

Reply:
Synced offline with Chenxi. I cannot use GetIpAddressFilePath to refer to the hostip.txt in line 60 since the path does not get resolved. It just appends the path to the AZ_BATCH_JOB_PREP_WORKING_DIR. I tried using Resolve-Path as well as Environment.ExpandEnvironmentVariables() and none of them worked.

---
In reply to: [200714213](https://github.com/sharathmalladi/reef/pull/6#discussion_r200714213) [](ancestors = 200714213,200411277)